lib/asciidoctor/standoc/converter.rb

```diff
@@ -89,11 +89,26 @@ def html_doc_path(file)
       end
 
       def flavor_name
```

Comment:
@opoudjis @ronaldtse maybe you can suggest a better way here?

Reply:
I don't have a better at the moment, but this is a good reminder that we need to update the interface...